netci.groovy

```diff
@@ -641,8 +338,13 @@ def supportedFullCyclePlatforms = ['Windows_NT', 'Ubuntu14.04', 'OSX']
             Utilities.addArchival(newJob, archiveContents)
             // Set up triggers
             if (isPR) {
-                // Set PR trigger.
-                Utilities.addGithubPRTriggerForBranch(newJob, branch, "Innerloop ${os} ${configurationGroup} Build and Test")
+                // Set PR trigger, we run Windows_NT, Ubuntu 14.04, CentOS 7.1 and OSX on every PR.
+                if ( os == 'Windows_NT' || os == 'Ubuntu14.04' || os == 'Centos7.1', os == 'OSX' )
```

Comment:
This is known as the syntax error operator.  You can use it to introduce a syntax error into otherwise valid groovy.  ;-).

I can't actually build this stuff locally and I also don't know how to submit a private test run, so you get to see my mistakes in real time.  Fixed.